### DISCUSSION TITLE
merge changes into the master branch

### DISCUSSION THREAD

**Discussion by ajoe** [ct-Open-Source/tuya-convert#1136] [General]
Hi Colin, 

could you do us a favor and either delete the old branches like master, python and so on or merge everthing from development into master? If you want I also can do that. 
I wasted some time just to see that I was in the wrong branch and master is not really a used branch anymore.
With this link you can do the merge

https://github.com/ct-Open-Source/tuya-convert/compare/master...development

thanks a lot,
Jochen

**Comment by kueblc** [comment]
You're absolutely right, master has fallen quite a bit behind development and a release is long overdue. Sorry about the trouble and thank you for bringing it back to my attention.

**Reply by ajoe** [reply]
I hope you had a beer or coffee ;-)
